SK.WebApp/src/services/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { ApiAdresses } from "./consts"
+
+export const getUrlWithCulture = (address: string, culture: string): string => {
+    return ApiAdresses.BASE + address.replace("{culture}", culture);
+};
```

Comment:
Whoa, nice use of `String.replace` - I wouldn't have thought of that!

Please use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). `+` is fine for numbers, when used in context of strings - it might lead to confusion.

My proposition:

```ts
// SK.WebApp/src/api/getEndpointWithLanguageCode.ts
import { anyEndpoint, endpoint } from './endpoint';
import { anyLanguageCode } from '../language';

interface getEndpointWithLanguageCodeArgs {
  endpointKey: anyEndpoint;
  languageCode: anyLanguageCode;
}

export const getEndpointWithLanguageCode = ({
  endpointKey,
  languageCode,
}: getEndpointWithLanguageCodeArgs) =>
  endpoint[endpointKey].replace('{languageCode}', languageCode);
```

Reply:
I refactored this